README.md

```diff
@@ -8,6 +8,14 @@ Originally I wrote a small plugin because I was tired seeing coffins instead of
 
 ![Screenshot](http://i.imgur.com/Xdmv3JH.png)
 
+## Installation ##
+All file paths are relative to your dwarf fortress installation directory  
+1. Override any conflicts presented by the following steps.
+2. Extract all .dll/.so/.dylib files to `hack/plugins`. If your package includes multiple sets of these libraries, make your best guess as to which is newest (betas tend to come after alphas, higher numbers imply newer versions, etc.) and use that one.
```

Comment:
Appreciate the credit, but yeah those directions were less 'production ready' and more 'flying by the seat of my pants'.